netbox/circuits/models.py

```diff
@@ -173,7 +173,11 @@ class Circuit(ChangeLoggedModel, CustomFieldModel):
 
     class Meta:
         ordering = ['provider', 'cid']
-        unique_together = ['provider', 'cid']
+        constraints = [
+            models.UniqueConstraint(
+                fields=['provider', 'cid'], name='circuit_provider_cid_key'
```

Comment:
Per @lampwins' note on the naming standard, the full table name is `dcim_circuit`. We probably want to keep the prefix for all the constraint names to be consistent with Django's stock approach.

Reply:
We can do this; so should we use the table name instead of the model name? I used the latter, but I’m okay with using the former.